ait/dsn/plugins/AOS_FEC_Check.py

```diff
@@ -0,0 +1,49 @@
+from ait.core.server.plugins import Plugin
+from ait.core import log
+from ait.dsn.sle.frames import AOSTransFrame, AOSDataFieldType
+from binascii import crc_hqx
+
+class AOS_FEC_Check(Plugin):
+    '''
+    Check if a AOS frame fails a Forward Error Correction Check
+    '''
+    def __init__(self, inputs=None, outputs=None, zmq_args=None, **kwargs):
+        super().__init__(inputs, outputs, zmq_args)
+        self.bytes_from_previous_frame = None
+        self.crc_func = crc_hqx
+        self.log_header = "AOS_FEC_CHECK =>" 
```

Comment:
amazing, fixes the issues we've had tying log messages to specific modules

Reply:
Yeah! 
Usually you would do:

`logger = logging.getLogger(__name__)`

Then your log header would be the actual function that's executing instead.

But...you know how it is...